src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -115,30 +131,73 @@ class TizenControls extends Controls {
             }
```

Comment:
Can `handleKeydown` become a private methods using `this.api`? (and a `apiEnabled` check)? That would prevent this from potentially leaking of still working if the listener wasn't removed properly but `this.api` was cleared.

Reply:
This method also uses `model` to get the state of the player. Not sure if it's also worth extracting model into a property as well, or creating a const that grabs the model state and calls the private `handleKeydown` instead.

Reply:
I'd say make it a property and clear api and model props on disable. Try to stick to using the view model everywhere and only accessing viewModel.player when you need to directly access a player model property (I saw the player model being passed in to another instance which could be a point of confusion since it's not consistent with other view components),